docs/components/connectors/out-of-the-box-connectors/microsoft-teams.md

```diff
@@ -70,6 +70,18 @@ Visit [Microsoft Teams Access Token](https://learn.microsoft.com/azure/active-di
   - [How to get access on behalf of a user](https://learn.microsoft.com/en-us/graph/auth-v2-user)
   - [How to get access without a user](https://learn.microsoft.com/en-us/graph/auth-v2-service)
 
+### Refresh Token type authentication
+
+For a **Refresh Token** type authentication, take the following steps:
+
+1. Click the **Refresh Token** connection type in the **Authentication** section.
+2. Set **Refresh Token** to `Refresh Token`. See more [how to get refresh token](https://learn.microsoft.com/en-us/graph/auth-v2-user)
+3. Set **Tenant id** to `Tenant ID`. Your Microsoft Teams tenant ID is a unique identifier. See more on [how to find a tenant ID](https://learn.microsoft.com/en-us/azure/active-directory/fundamentals/active-directory-how-to-find-tenant).
+4. Set the **Client ID** field: the application ID that the [Azure app registration portal](https://go.microsoft.com/fwlink/?linkid=2083908) assigned to your app.
+5. Set the **Secret ID** field: the client secret that you created in the app registration portal for your app.
+
+![refresh token](../img/connectors-microsoft-teams-refresh-token.png)
```

Comment:
On a screenshot, you have a CTA (call to action) "See connector documentation". Could you add please an URL link to documentation?